doc/classes/RenderingServer.xml

```diff
@@ -4272,6 +4272,9 @@
 		</constant>
 		<constant name="LIGHT_PARAM_TRANSMITTANCE_BIAS" value="19" enum="LightParam">
 		</constant>
+		<constant name="LIGHT_PARAM_INTENSITY" value="20" enum="LightParam">
+			The intensity of the light source measured in Lumens. Typically used when [member ProjectSettings.rendering/lights_and_shadows/use_physical_light_units] is [code]true[/code] to determine the total energy of a given light.
```

Comment:
This isn't quite accurate, intensity can be measured in lumens (when the light is a spot light or an omni light) or lux (when the light is a directional light). Additionally, the setting only has an effect when ``ProjectSettings.rendering/lights_and_shadows/use_physical_light_units`` is true

For reference, the description in Light3D is:

> Constant for accessing [member light_intensity_lumens] and [member light_intensity_lux]. Only used when [member ProjectSettings.rendering/lights_and_shadows/use_physical_light_units] is [code]true[/code].

Reply:
thank you for this insight!